### ISSUE TITLE
migrate back to Stripe

### ISSUE THREAD

**Issue by seanlinsley** [gratipay/gratipay.com#3245]
http://blog.balancedpayments.com/shutting-down

>We started building a payments service for marketplaces in 2011. We were shocked with the state of payments at the time, and the pains a marketplace had to go through. Despite the many challenges in the industry, we’re proud of how much we’ve been able to help marketplaces thrive over the last four years.

>Unfortunately, we haven’t been able to reach the escape velocity necessary to be a large, innovative, independent player in the payments space and have decided not to continue building Balanced.

>It’s important to us to identify a graceful transition for our customers, employees and investors – and honor our existing agreements. To that end, we reached out to Stripe and have been working with them to provide a smooth path forward for our customers. We respect how Stripe has been able to innovate, grow, and become the best solution for marketplaces. Stripe has agreed to honor our existing pricing agreements and work closely with us to seamlessly migrate card, bank account, and seller identity data.

>Our full API and dashboard will be available over the next 90 days (until June 11, 2015), and support for issuing refunds, querying transactions, and fighting chargebacks will continue for the next nine months (until October 9, 2015).

>We realize this is sudden and we’re sorry for any inconvenience this causes. The full Balanced team will stay on for the 90-day migration period and a portion of the team will continue beyond then to make sure we’re able to provide you with the level of support you need. We appreciate you working with us and want to do everything we can to minimize the impact on your business.

>To learn more about the migration, what it means to you, and the steps required to make the switch, go to balancedpayments.com/stripe. You’ll be able to sign into your dashboard and begin the process to migrate your data to Stripe.

**Comment by seanlinsley** [comment]
Migration instructions: https://www.balancedpayments.com/stripe

Hacker News: https://news.ycombinator.com/item?id=9200017

**Comment by clone1018** [comment]
Shit.

**Comment by colindean** [comment]
Well, then.

**Comment by seanlinsley** [comment]
Is it even possible for Stripe to support us? Trying to figure that out currently.

**Comment by seanlinsley** [comment]
It seems like the answer is yes. The differences being [invoicing](https://www.balancedpayments.com/stripe/faq#how-will-invoicing-change-):

> Balanced and Stripe invoice for fees differently. While Balanced invoices you for fees directly from your bank account at the end of each business day, Stripe deducts the fee from each transaction. If your pricing is 2.9% + 30¢ and you perform a $100 charge, Balanced would add $100 to your escrow and invoice you for the $3.20 fee. Stripe instead will add $96.80 ($100 - $3.20) to your Stripe balance, and not issue a separate invoice.

And [ACH transfers](https://stripe.com/us/marketplaces):

> Send transfers for 25¢ each. Transfers to your own bank account are always free.

And [underwriting](https://www.balancedpayments.com/stripe/faq#how-will-merchant-underwriting-change-):

> Stripe requires name and DOB for all customers before a payout is issued. Stripe may also ask for a physical address and the last 4 digits of the customer's social security number as the seller's volume increases. The verification works similarly to how it did with Balanced—except Stripe supports uploading a photo ID when the seller's information cannot be automatically verified, which will allow you to onboard more sellers more quickly.

Since we don't have the name, email address, or DOB of the typical Gratipay user, that last point could cause quite a bit of pain.

**Comment by whit537** [comment]
I've added the dates to the [calendar](http://inside.gratipay.com/appendices/calendar).

**Comment by Changaco** [comment]
Created the milestone https://github.com/gratipay/gratipay.com/milestones/Balanced%20shutdown for this and related issues.

**Comment by Changaco** [comment]
> Is it even possible for Stripe to support us?

We're going to hit one of the problems we had with Transpay: for international payouts they want to know the identity of the *sender*, but they don't allow *multiple* senders, and typically the money paid out from Gratipay comes from multiple patrons.

**Comment by Changaco** [comment]
https://stripe.com/docs/connect/special-case-transfers

> For compliance reasons, the vast majority of money moving around *must* be directly linked to an incoming charge

**Comment by rohitpaulk** [comment]
[IRC #stripe](https://botbot.me/freenode/stripe/2015-03-25/?msg=35018289&page=5)

**Comment by whit537** [comment]
Yeah, this is pretty risky for us. :-(

**Comment by Changaco** [comment]
> Yeah, this is pretty risky for us. :-(

For now I think we can use [the deprecated US-only API](https://stripe.com/docs/tutorials/sending-transfers) (similar to Balanced), but I think we should contact Stripe, explain our situation and ask them if there is a way for us to make international payments without linking each Transfer to a specific Charge. It's not that we don't know where the money comes from, it's just that it usually comes from multiple charges.

**Comment by whit537** [comment]
> the deprecated US-only API (similar to Balanced)

What about the [beta managed accounts feature](https://stripe.com/docs/connect/managed-accounts)?

**Comment by whit537** [comment]
> What about the beta managed accounts feature?

Sounds like that might be new? From Matin in bcc to private email:

>Hi everyone,
>
Over the last two weeks our customers have migrated over a million cards and a hundred-thousand bank accounts to Stripe. We're working hard on simplifying the process for you, and I'm sending you this email to give you a few updates.
>
**Pricing**
 1. Stripe will continue to honor your existing pricing for charging cards and payouts. The 0.5% managed account pricing announced by Stripe a week ago is for new Stripe customers, but Balanced customers will keep their 25¢ payouts pricing.
 1. Stripe's pricing for ACH debits for all Balanced customers is 25¢ / transaction. For most of you, that's less than what you pay today. :-)
>
**Stripe Connect**
Last week, Stripe launched Stripe Connect (https://stripe.com/connect), which has been entirely rebuilt to meet the specific needs and workflows of marketplaces. With its new functionality, Stripe Connect simplifies the transition for you further. Here are a few things you should know about Connect:
>
 1. Customers on Balanced are migrated to Customers on Stripe.
 1. Customers on Balanced that have received a payout are also migrated to Managed Accounts (Stripe's terminology for seller) on Stripe without requiring additional verification. As a heads up, at some point Stripe will want to verify your migrated sellers who do not already have name, address, DOB, and SSN—both so they can send 1099Ks and to meet compliance requirements. Stripe will reach out well in advance of any requirements and will work with you on gathering this information. 
 1. Stripe Connect has the concept of Managed and Standalone Accounts. Managed Accounts are the white-labeled version (just like Balanced)—allowing you to keep the same seller signup flow.
 1. We've created a full migration guide to go into more detail: https://balancedpayments.com/stripe/migration-guide
>
**ACH Debits**
Stripe has an ACH Debits solution in private beta. Here are a few things you should know about Stripe's ACH Debit solution:
>
 1. Going through the Balanced data migration process automatically enters you into the private beta!
 1. If you signed up for Stripe directly, you can email sales@stripe.com to give you access to the private beta.
 1. Integration documentation: https://stripe.com/docs/guides/ach-beta
>
**Improvements on the data migration**
>
 1. There have been several performance enhancements to make sure your data is migrated faster.
 1. Originally, only verified / underwritten customers on Balanced were migrated on Stripe as Accounts. We now create an Account on Stripe for any customer that has ever received a payout.
 1. There is now a button in the Balanced dashboard to download a CSV file that shows the mapping between Balanced ID's and Stripe ID's
>
**Misc**
>
There are a few other questions we've seen that I wanted to cover in this email. Most of these questions are addressed in our FAQ, which we actively update: https://www.balancedpayments.com/stripe/faq
>
 1. Stripe gives you the ability control when a buyer is charged and a seller is paid. Balanced called this functionality "escrow". More detail: https://www.balancedpayments.com/stripe/faq#escrow-functionality
 1. We've received a lot of questions about the ability to preload / top up your balance. Stripe provides similar functionality: you have the ability to keep proceeds from prior transactions in your Stripe account balance to subsidize or provide future credits to sellers. Stripe will also provide a one time preload when you start. You can contact sales@stripe.com to perform the initial funding of your balance.
 1. Balanced and Stripe are both working hard on providing you support during the migration.
  - If you have questions about the data migration and differences between Balanced and Stripe, please email support@balancedpayments.com.
  - If you have Stripe-specific questions like international support and different payment methods, please head over to https://support.stripe.com/email.
  - If you have questions on the technical migration, you can contact Balanced on #balanced and Stripe on #stripe on Freenode (IRC).
>
Feel free to also email me directly at matin@balancedpayments.com.
>
Best,
Matin

**Comment by rohitpaulk** [comment]
> What about the beta managed accounts feature?
> Sounds like that might be new? From Matin in bcc to private email:

The same problem, @whit537 - https://github.com/gratipay/gratipay.com/issues/3245#issuecomment-86093590

> For compliance reasons, the vast majority of money moving around must be directly linked to an incoming charge (either by having been created on the connected account or via use of the destination parameter).

**Comment by whit537** [comment]
@rohitpaulk Gotcha, seeing now that you mentioned managed accounts in [IRC](https://botbot.me/freenode/stripe/2015-03-25/?msg=35018316&page=5). Hrm ...

**Comment by whit537** [comment]
> It's not that we don't know where the money comes from

Do we know where the money comes from? #3289 :-(

**Comment by whit537** [comment]
Sent to sales@stripe.com:

>Greetings!
>
Balanced customer here, evaluating our options. We are a marketplace for recurring (weekly) gift tips and voluntary payments:
>
https://gratipay.com/
>
We specialize in micropayments: we charge Alice $10 per week and split that up between 10 recipients, who may be friends or colleagues she's tipping, or pay-what-you-want businesses she's supporting. The initial feedback we're getting in IRC is that this would be a special case for Stripe to support.
>
Therefore, I'm writing to start a conversation about Stripe underwriting us. Quick snapshot of our business:
>
 - ~3,000 active users
 - ~4,750 charges/mo
 - ~800 payouts/mo
 - ~$40,000/mo total volume
 - ~$10 avg charge
 - ~$40 avg payout
>
Two further initial points:
>
 - Our chargeback rate is quite low: 0.016% (9 / 57,138) over three years.
 - Stripe has been [a customer of ours](https://gratipay.com/stripe/). :-)
>
Are you willing to underwrite us?
>
>
chad
>
Chad Whitacre, Founder, [Gratipay](https://gratipay.com/)
chad@zetaweb.com
+1-412-925-4220 (cell)

**Comment by whit537** [comment]
https://twitter.com/whit537/status/583358661168406529

**Comment by Changaco** [comment]
> Do we know where the money comes from? #3289 :-(

I've already answered that in https://github.com/gratipay/gratipay.com/issues/3289#issuecomment-86484389.

**Comment by whit537** [comment]
> I've already answered that in #3289 (comment).

Thanks.

**Comment by whit537** [comment]
> We're looking into it and will get back to you shortly.

https://twitter.com/stripe/status/583714168542486528

**Comment by whit537** [comment]
Stripe sales in private email:

> I'm digging into this now: you're right that this is a special case that we haven't supported thus far. I'd like to have a couple of discussions internally before I give you a solid yes or no, but will let you know ASAP. Two questions for you:
>
1) Are your bank payouts US-only right now?
2) I see you're syndicating everything to Github - would it be easier if I move the conversation there? I of course would not reveal any personal info, but it sounds like it would save you some copy/pasting.

My reply:

> Thanks for getting back to me, Brian. Short answer: yes, our bank payouts are US-only right now. Longer answer: we have a lot of pressure to support first-class non-US payouts―it's our [top issue](https://github.com/gratipay/gratipay.com/issues?q=is%3Aissue+is%3Aopen+sort%3Acomments-desc)―and we currently use PayPal and to a lesser extent bitcoin as bandaids.
>
And yes, discussing on GitHub would be wonderful, thanks! :-)

**Comment by bkrausz** [comment]
I'm the Brian on the private email, moving the conversation here. Will update once I have more information on our ability to support Gratipay's model.

**Comment by whit537** [comment]
!m @bkrausz :-)

**Comment by bkrausz** [comment]
Sorry for the delay responding here: we're working to try and find a way that Gratipay can work within our current regulatory framework. I will have an answer for you tomorrow.

One question beforehand: do you have an idea of what % of payments are paid out directly (i.e. credit card charge to bank account) vs. effectively balance transfers?

**Comment by Changaco** [comment]
/me starts writing a script to compute that

**Comment by Changaco** [comment]
Result:

Direct payments from a credit card to a bank account:

| count |  sum   |
|-------|--------|
|    43 | 770.00 |

Out of all credit card charges:

|  status   | count |   sum   |
|-----------|-------|---------|
| succeeded |   448 | 7140.15 |
| failed    |   648 | 3086.89 |

**Comment by Changaco** [comment]
Data from last week's payday, amounts in US dollars of course.

**Comment by whit537** [comment]
@Changaco Could you post the script you came up with? Strictly speaking, we don't have "[d]irect payments from a credit card to a bank account." I can guess how you interpreted that for our case, but seeing the script would clarify.

Percentage-wise we're looking at 10% if we only count successful charges, or 4% if we take all charges as the base.

**Comment by Changaco** [comment]
@whit537 https://gist.github.com/Changaco/35eaab12b556da04c00d

**Comment by bkrausz** [comment]
Unfortunately with Gratipay's current model we would be unable to support them on Stripe.

The crux of the problem is the stored value: holding funds in a balance that can then be redistributed, in our opinion, qualifies as money transmission, which we cannot support. There are other compounding factors that would make it hard to reason around this property, for example the lack of a concrete product or service being provided.

I'm sorry for this outcome: there was a fair bit of discussion internally, and we tried to work out a framework in which we could support Gratipay, and at least for 2015 there is no regulatory footing under which we can do so.

**Comment by whit537** [comment]
Thanks for trying, @bkrausz. I've dug up the private email conversation I had with Stripe the last time this happened, back in 2012, because I seemed to remember something about a volume threshold above which our model would be a problem. I agreed back then not to publicly discuss the details (see public notes on https://github.com/gratipay/gratipay.com/issues/67), but the tl;dr is that Gratipay is well above that threshold now.

I've reticketed getting a money transmitter license as #3321.

Best wishes, Stripe. :-)
